scripts/ios/helper.js

```diff
@@ -58,6 +58,13 @@ function ensureUrlSchemeInPlist(urlScheme, appPlist){
 // Public functions
 module.exports = {
 
+    /**
+     * Value from
+     * https://github.com/apache/cordova-ios/blob/rel/7.0.0/templates/project/__PROJECT_NAME__.xcodeproj/project.pbxproj#L120
+     * TODO: Update after release cordova-ios v8
+     */
+    appPBXGroup: '29B97315FDCFA39411CA2CEA',
```

Comment:
Wonder is there's a way we can derive this programmatically?
As I think it will be a problem if it supports only `cordova-ios@7` OR `cordova-ios@8` 
Because I don't think we can't tell which platform version is installed from the Cordova context passed to hook scripts so we won't know which value to use.
Not everyone will update to `cordova-ios@8` immediately (or at all).

Reply:
I've found a programmatically way. 

Need some time to debug

Reply:
Great, thanks for contributing this